make_ids.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import csv
+import json
+import os
+import sys
+
+
+def format_entities_as_list(entities):
+    for i, entity in enumerate(entities, 1):
+        yield (unicode(i), json.dumps(entity["terms"]))
+
+
+def generate_entities(fobj):
```

Comment:
Added a docstring.

fobj is a common idiomatic name in python projects I've worked on for "file object" - I agree it needs explaining, though.